packages/search-spa/src/environments/environment.ts

```diff
@@ -0,0 +1,4 @@
+export const environment = {
+   API_URL: process.env.API_URL,
+   production: true,
```

Comment:
For non-production builds.

```suggestion
   production: false,
```

Reply:
updated